Tests/Marketplace/upload_packs.py

```diff
@@ -342,7 +341,7 @@ def upload_index_to_storage(index_folder_path: str, extract_destination_path: st
             # Store index.json in CircleCI artifacts
             shutil.copyfile(
                 os.path.join(index_folder_path, f'{GCPConfig.INDEX_NAME}.json'),
-                os.path.join(artifacts_dir, f'{GCPConfig.INDEX_NAME}.json'),
+                os.path.join(artifacts_dir, f'{GCPConfig.INDEX_NAME}-{"id_set" if id_set else "graph"}.json'),
```

Comment:
Is this by design or just for testing?
If its by design, then need to fix when reading this file as well.

Reply:
I'm not sure when we read this file from Gitlab artifacts, but to be on the safe side I kept the same name if we're using the id_set